File: Ex16.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    // Inserção das variáveis

    int NumeroX, NumeroY, NumeroZ, Confirmacao;

    // Inserção do primeiro valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroX);

    
    // Inserção do segundo valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroY);
    
    
    // Inserção do terceiro valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroZ);

    //Confirmaçao do Triângulo
    Confirmacao= NumeroX+NumeroY;


    //Comparação entre valores
    if (Confirmacao>NumeroZ)
    {
        if (NumeroX==NumeroY)
        {
            if (NumeroX==NumeroZ)
            {
                printf("A figura formada e um triangulo equilatero");
            }
            else {
                printf("A figura formada e um triangulo Isoceles");
            }
        }
        else {
            if (NumeroY==NumeroZ)

            {
                printf("A figura formada e um triangulo Isoceles");
            }
            else {
                printf("A figura formada e um triangulo escaleno");
            }
            
        }
            
    }
    else{
    
        printf("A figura formada nao e um triangulo");

    }
        
    
    
    
    return 0;
}

File: Ex16.py
v1 = float(input("Insira um numero qualquer: "))
v2 = float(input("Insira um outro numero qualquer: "))
v3 = float(input("Insira um terceiro numero qualquer: "))

vl = v1 + v2

if vl > v3:
    if v1 == v2 and v1 == v3:
        print("Os tres numeros formam um triangulo equilátero")

    elif v1 == v2 or v1 == v3 or v2 == v3:
        print("Os tres numeros formam um triangulo isosceles")
    
    else:
        print("Os tres numeros formam um triangulo escaleno")

else:
    print("Os tres numeros nao formam um triangulo")
